app/views/posts/show.html.erb

```diff
@@ -1,17 +1,23 @@
-<p>post # <%= @post.id %> by <%= @user.name %></p>
-<p>comments: <%= @post.commentsCounter%> likes: <%= @post.likesCounter %></p>
-<h1><%= @post.title %></h1>
-<p><%= @post.text %></p>
-<h3>Comments:</h3>
-<ul>
-  <% @post.comments.each do |comment|%>
-    <li>
-      <p>posted by <%= comment.user.name %></p>
-      <p><%= comment.text %></p>
-    </li>
-  <% end %>
-</ul>
-
-<br/>
-<br/>
-<br/>
+<div class='container'>
+<div>
+  <div class='post'>
+    <div class='post-header'>
+      <p>post # <%= @post.id %> by <%= @user.name %></p>
+      <div class='comment-like'>
+      <p>comments: <%= @post.commentsCounter%></p>
+      <p>likes: <%= @post.likesCounter %></p>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add default values(e.g.: 0) on your `commentsCounter` and `likesCounter` for zero comment and like respectively:
```ruby
      <p>comments: <%= @post.commentsCounter || 0 %></p>
      <p>likes: <%= @post.likesCounter  || 0 %></p>
```